### PR TITLE
D3ASIM-2181: Disable the redis queue print -Show D3A Logging Thru Logger

### PR DESCRIPTION
**Pull request by psurender**
Please Review

### CODE REVIEW

File: src/d3a/d3a_core/d3a_jobs.py
Comment (faizan2590) [inline src/d3a/d3a_core/d3a_jobs.py:141]:
```suggestion
            name='simulation.{}.{:%s}'.format(getpid(), now()), log_job_description=False
```
Please follow flake8 in order of travis build to pass.

Also curious to know if you used git cli to commit or a git gui tool because git cli always pre-catch it while committing.

**Review by faizan2590 — COMMENTED** [review]
Have left one comment, will LGTM once addressed.

File: src/d3a/d3a_core/d3a_jobs.py
Comment (psurender) [reply src/d3a/d3a_core/d3a_jobs.py:141]: while commiting its not identified this actually for d3a-web its identifying, ok will remove this space

Comment (spyrostz) [reply src/d3a/d3a_core/d3a_jobs.py:141]: Actually it is vice-versa: d3a should include `flake8` checks, while d3a-web does not. Having said that, what I usually do to avoid such problems is to run `tox -e travis` locally before I open a PR to make sure that my changes pass all required syntactic, unit and integration tests.

**Review by spyrostz — COMMENTED** [review]
A general comment, this PR indeed disables the logs from the Redis queue. What I do not see here though is where are the corresponding logs from d3a printed. The idea behind this ticket AFAIK is to not print the scenario in compressed binary format (which is sent by the redis queue) but to print it once it has been uncompressed and in string format. If you can point me to the corresponding log then all is good and I will LGTM, otherwise such a log needs to be added.

File: src/d3a/d3a_core/d3a_jobs.py
Comment (psurender) [reply src/d3a/d3a_core/d3a_jobs.py:141]: Spyros ,Sure , I will do that

**Comment by psurender** [comment]
> needs

Yes Spyros, me and Faizan was discussed on this and I have analysed on this, but I dont see option to convert certain set of logs  , I have used decompress option with format of utf-8, but still conversion is not happening, This will take some time to identify.

File: src/d3a/d3a_core/d3a_jobs.py
Comment (spyrostz) [inline src/d3a/d3a_core/d3a_jobs.py:44]: Please revert this to error

File: src/d3a/d3a_core/d3a_jobs.py
Comment (spyrostz) [inline src/d3a/d3a_core/d3a_jobs.py:50]: Change these logs to error accordingly

File: src/d3a/d3a_core/d3a_jobs.py
Comment (spyrostz) [inline src/d3a/d3a_core/d3a_jobs.py:50]: Travis is complaining about this line, that it is too long. I think that there is something wrong with the git precommit hooks in your local machine, otherwise you would not be able to commit this change.

As for the actual fix, if you split this line in 2 lines the problem will resolve.

**Review by spyrostz — COMMENTED** [review]
LGTM once Travis passes

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by faizan2590 — APPROVED** [review]
LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**
